pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -244,19 +244,21 @@ self: super: {
   # base bound
   digit = doJailbreak super.digit;
 
-  # https://github.com/jwiegley/hnix/issues/98 - tied to an older deriving-compat
-  hnix = (overrideCabal super.hnix (old: {
+  # dontCheck: Can be removed once https://github.com/haskell-nix/hnix/commit/471712f is in (5.2 probably)
+  #   This is due to GenList having been removed from generic-random in 1.2.0.0
+  # doJailbreak: Can be removed once https://github.com/haskell-nix/hnix/pull/329 is in (5.2 probably)
```

Comment:
This has been merged.

Reply:
Neat! Now just needs a new hackage release and a nixpkgs lts update to remove the hacks :)

Reply:
I'll do this once we have the Travis builds for hnix working again.